.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: objective-c
 osx_image: xcode7.1
 
+before_install:
+  - brew update
+  - brew install swiftlint 
```

Comment:
Maybe, I'm using the travis doc to setup this.
http://docs.travis-ci.com/user/osx-ci-environment/#Homebrew